py/dml/dmlc.py

```diff
@@ -513,6 +519,15 @@ def show_illegal_attributes(option, opt, value, parser):
 
     dml.globals.state_change_dml12 = options.state_change_dml12
 
+    dml.globals.enable_testing_features = options.enable_testing_features
+
+    if dml.globals.enable_testing_features:
+        prerr("dmlc: ***Features for internal testing are enabled!!! "
+              + "These are near-guaranteed to NOT SURVIVE in their current "
+              + "form. They are COMPLETELY UNSUPPORTED. "
+              + "The DMLC developers WILL NOT respect their use. "
+              + "NEVER enable this flag for any kind of production code!!!***")
```

Comment:
nitpick: we usually emit `***` in a different style, `*** tool: message`. Another options could be to make this a `WEXPERIMENTAL`.

Does not block merge.

Reply:
Fairly certain `*** dmlc: ` would cause our TC builds to fail.